src/location-registry/controllers/helpers.py

```diff
@@ -0,0 +1,26 @@
+from flask import Blueprint, request, jsonify
```

Comment:
- Could we try to use more descriptive names for our helper files? What does this helper file do?
- And why is this helper file in the controllers directory? Could we move it to the helpers directory?

Reply:
helper file has been removed and its functionality moved to the main controller file